[Package Request]: FileShows version:1.0.0

huanguan1978 opened the pull request:
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---
Download Page Url: https://webpath.iche2.com/app/fileshows/download_en.html
Publisher: Heshan Aisipai E-commerce Co., Ltd.
Package Name: FileShows
Description: A file searcher with glob patterns and file streams
Package Version: 1.0.0
Installer Url: https://webpath.iche2.com/release/FileShows-1.0.0-Setup.exe
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/310323)

huanguan1978 committed:
[Package Request]: FileShows version:1.0.0

Download Page Url: https://webpath.iche2.com/app/fileshows/download_en.html
Publisher: Heshan Aisipai E-commerce Co., Ltd.
Package Name: FileShows
Description: A file searcher with glob patterns and file streams
Package Version: 1.0.0
Installer Url: https://webpath.iche2.com/release/FileShows-1.0.0-Setup.exe

stephengillie commented:
Verify:
<img width="1010" height="709" alt="image" src="https://github.com/user-attachments/assets/6dd04135-7082-4238-b803-12c2338ba4a9" />
<img width="385" height="151" alt="image" src="https://github.com/user-attachments/assets/4df06ec7-e3b7-44d0-9e34-1e1474a0f78b" />

huanguan1978 committed:
Merge branch 'microsoft:master' into master

huanguan1978 committed:
Updated: Inline Contextual Help for Quick Apps

Updated InstallerSha256 for the installer URL.
Updated: Inline Contextual Help for Quick Apps

huanguan1978 commented:
@stephengillie @wingetbot Thank you for the update.

I've verified that both the **Download Page URL** and **Installer URL** (`https://webpath.iche2.com/app/fileshows/download_en.html` and `https://webpath.iche2.com/release/FileShows-1.0.0-Setup.exe` respectively) are **publicly accessible** via standard web browsers.

Regarding domain validity and publisher alignment:
*   **Publisher:** Heshan Aisipai E-commerce Co., Ltd. (`鹤山市爱思派电子商务有限公司`) is the **registered legal entity** for `iche2.com`.
*   `webpath.iche2.com` is a **direct subdomain** of `iche2.com`.
*   The provided screenshot details the **official ICP filing** for `iche2.com`. This is a **mandatory government registration in China** that unequivocally validates `iche2.com`'s **corporate ownership and operational legitimacy** by Heshan Aisipai E-commerce Co., Ltd.

Given this, the domain `webpath.iche2.com` is demonstrably aligned with our official publisher entity.

Could you specify any **technical access errors or required supplemental documentation** to fulfill the domain/publisher validation for a Chinese ICP-filed entity?

stephengillie commented:
Hi @huanguan1978,

Sorry for the delay in responding. I have a large volume of other PRs that also require my attention. This PR shares a large number of details with #202507, including publisher details.
